plugins/modules/azure_rm_aks.py

```diff
@@ -657,6 +697,10 @@ def create_addon_profiles_spec():
     ssh_key=dict(type='str', no_log=True, required=True)
 )
 
+windows_profile_spec = dict(
+    admin_username=dict(type='str', required=True),
+    admin_password=dict(type='str', required=True),
```

Comment:
admin_password could be a secret
```suggestion
    admin_password=dict(type='str', no_log=True, required=True),
```